source/user-portal/assets/favRecipe.js

```diff
@@ -76,24 +77,44 @@ class FavRecipe extends HTMLElement {
 
         const recipe = document.createElement("article");
 
-        const recipeTitle = document.createElement("a");
-        recipeTitle.setAttribute('href',"");
-        recipeTitle.setAttribute("class",'title');
+        let recipeTitle = document.createElement("p");
 
-        const recipeImg = document.createElement('img');
-        const timeCircle = document.createElement('div');
-        timeCircle.setAttribute("class", 'time');
+        let recipeImg = document.createElement("img");
 
-        const timeNumb = document.createElement('p')
-        timeNumb.setAttribute("class", 'recipe-time');
+        let timeCircle = document.createElement("div");
+        timeCircle.setAttribute("class", "time");
 
-        timeNumb.innerHTML = "0";
+        let timeNumb = document.createElement("p");
+        timeNumb.setAttribute("class", "recipe-time");
 
-        recipeImg.setAttribute("src",'images/sadburger.gif');
+        if(data == "Favorite more recipes!"){
+            // Adding Title
+            const text = document.createTextNode("Please favorite more recipes!");
+            recipeTitle.appendChild(text);
 
-        recipeTitle.innerHTML = data;
+            // Adding Img
+            recipeImg.setAttribute("src", 'images/sadburger.gif');
 
-        
+            // Adding Time
+            timeNumb.innerHTML = "0";
+        }
+        else{
+
+            let json = JSON.parse(localStorage.getItem(data));
+
+            // Adding Title
+            const text = document.createTextNode(data);
+            recipeTitle.appendChild(text);
+
+            // Adding Img
+            let value = searchForKey(json, 'thumbnailUrl');
+            recipeImg.setAttribute("src", value);
+
+            // Adding Time
+            value = searchForKey(json, 'totalTime');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8436327)
```suggestion
            value = searchForKey(json, "totalTime");
```